benchmark/timings.cpp

```diff
@@ -332,6 +335,42 @@ int main(int argc, const char ** argv)
     forwardKinematics(model,data,qs[_smooth],qdots[_smooth], qddots[_smooth]);
   }
   std::cout << "Second Order Kinematics = \t"; timer.toc(std::cout,NBT);
+
+
+  timer.tic();
+  SMOOTH(NBT)
+  {
+    updateFramePlacements(model,data);
+  }
+  std::cout << "Update All Frames Placement = \t"; timer.toc(std::cout,NBT);
+
+  timer.tic();
+  SMOOTH(NBT)
+  {
+    updateFramePlacement(model,data, fids[_smooth]);
```

Comment:
This one is not relevant nether, it would be the just the evaluation of the multiplication of two rigid transforms

Reply:
I agree, I simply wanted to see the difference between doing it for a frame vs doing it for full tree. I'll remove it.